otherlibs/unix/unix.mli

```diff
@@ -543,6 +543,28 @@ val map_file :
   validation fails.
   @since 4.06.0 *)
 
+val flush_mapped_file :
+  ('a, 'b, 'c) CamlinternalBigarray.genarray -> bool -> unit
```

Comment:
I'm in favor of labeling the bool argument (even in Unix); this makes the client code more readable (the meaning of the bool is really not obvious without looking at the documentation).